PowerShellTools/Resources.resx

```diff
@@ -405,4 +405,16 @@ Would you like to download now?</value>
   <data name="AttachErrorTitle" xml:space="preserve">
     <value>Error Attaching to Process</value>
   </data>
+  <data name="CleanupErrorMessage" xml:space="preserve">

```

Comment:
Message was changed in 5328051 to: PowerShell Tools is unable to detach from the process and may now be in an unstable state. You may need to restart Visual Studio.